flocx_market/objects/bid.py

```diff
@@ -0,0 +1,40 @@
+import flocx_market.db.sqlalchemy.api as db
+
+
```

Comment:
You probably want a class here; this should look similar to: https://github.com/CCI-MOC/esi-leap/blob/master/esi_leap/objects/offer.py#L24-L37

Reply:
Thanks that was my next step, after the simpler one starts working, now I will try to follow the ESI Leap code :)